src/transformers/modeling_auto.py

```diff
@@ -243,6 +249,7 @@
         (ElectraConfig, ElectraForMaskedLM),
         (EncoderDecoderConfig, EncoderDecoderModel),
         (ReformerConfig, ReformerModelWithLMHead),
+        (ProphetNetConfig, ProphetNetModel),
```

Comment:
Is this correct? No need for `WithLMHeadModel`?

Reply:
Deleted this line